README.md

```diff
@@ -81,13 +81,13 @@ SNAPLET_PROJECT_ID=// Project ID found in Snaplet project settings.
 ```
 
 ## How it works
-<img width="800" alt="How it works" src="https://user-images.githubusercontent.com/39437696/190385467-0d704a59-7a89-4dba-a68b-eb8b2ca7c1fe.png">
+<img width="800" alt="How it works" src="diagram.svg">
```

Comment:
I would suggest that it's better to keep these out of the repo and rather upload and link them via an issue.

Reply:
but I kinda think it's ok since this is an awesome diagram that's about architecture, can we rename to "architecture.svg?